YubiKit/YubiKit/Helpers/Additions/YKFNSDataAdditions.m

```diff
@@ -248,7 +248,14 @@ - (BOOL)ykf_containsRange:(NSRange) range {
 @implementation NSData(NSData_Base32Additions)
 
 + (NSData *)ykf_dataWithBase32String:(NSString *)base32String {
```

Comment:
Can you please add couple UTs for this method...which will should that correct string works and incorrect one returns nil